canonical-web-and-design/jaas-dashboard
# Render the config panel based on the URL.

## Done

- Update the url handling so that the config panel is rendered based on the URL and not part of the App panel.
- Fix sticky description bug from animation branch.

## QA

- Pull code
- Run dashboard
- Navigate to the config UI for an application
- Refresh the page, it should end up loading the config panel again.
- Make your browser vertically small, focus a config input and scroll the page, the config description should stick to the header.

## Details

Fixes https://github.com/canonical-web-and-design/jaas-dashboard/pull/845
